source/_integrations/tibber.markdown

```diff
@@ -56,6 +56,39 @@ You also get sensors for monthly consumption, monthly cost, and monthly peak hou
 If you have a Tibber Pulse it will also show the electricity consumption in real time.
 
 
+## Available Sensors
```

Comment:
Please don't list sensors by their entity IDs. They are not guaranteed and don't have to be the same on your system.

Looking at the naming use, it is based on an old naming scheme as well, so already incorrect.

Instead, please just describe the entities.

PS: Please avoid the use of tables. They are generally hard to read on mobile.

Reply:
Adjusted in accordance (hopefully)